src/game/core/explore.js

```diff
@@ -11,24 +10,27 @@ import {
 
 import { Observable } from 'rx'
 
-import models from '../models'
+import {
+  maps,
+  monsters,
+} from '../models'
+
 import { run } from './combat'
 
-export function randomMonster (mapId) {
-  const mapObj = models.maps.find(mapId)
-  const monsterPool = mapObj.monsters.reduce((acc, monster) => [
+export function randomMonster (monsters, pickMonsterFn=monsters.find) {
+  const monsterPool = monsters.reduce((acc, monster) => [
     ...acc,
-    ...Array.from({ length: monster.influence }).map(always(monster.id)),
+    ...Array.from({ length: monster.influence }, ()=> monster.id ),
```

Comment:
Well, my unique opinion is about coding-style that the linter will catch. In this case, `()=>` to `() =>`.